test/cases/orchestration.py

```diff
@@ -534,6 +543,10 @@ def nginx_test_config(self, nginx_conf_text, file_name):
         # -T            : test configuration, dump it and exit
         # -q            : suppress non-error messages during configuration testing
         # -c filename   : set configuration file (default: /etc/nginx/nginx.conf)
+        nginx_conf_text = nginx_conf_text.replace(
+            "load_module modules/ngx_http_datadog_module.so",
+            "load_module /test-conf/ngx_http_datadog_module.so",
+        )
```

Comment:
Per our discussion Thursday, please instead modify the actual test config files -- something like this:
```nginx
# "/test-conf" is a directory created by the docker build
# of the nginx test image. It contains the module, the
# nginx config, and "index.html".
load_module /test-conf/ngx_http_datadog_module.so;
```